py/torch_tensorrt/dynamo/conversion/impl/unsqueeze.py

```diff
@@ -18,12 +19,84 @@ def unsqueeze(
     input: TRTTensor,
     dim: int,
 ) -> TRTTensor:
+    from importlib.metadata import version
+
+    if version("tensorrt") < "10.7.0":
+        return unsqueeze_old(ctx, target, source_ir, name, input, dim)
     axes = get_trt_tensor(ctx, dim, f"{name}_axes")
     layer = ctx.net.add_unsqueeze(input, axes)
     set_layer_name(layer, target, name, source_ir)
     return layer.get_output(0)
 
 
+# old implementation for jetson due to IUnsqueezeLayer was not supported prior to 10.7.0
+def unsqueeze_old(
```

Comment:
Where does this get used in this PR?

Reply:
I think its better to have a different converter or something at a higher level so that its clear to people debugging they are using a handwritten implementation